packages/patternfly-3/patternfly-react/src/components/Overlay/__mocks__/mockOverlayManager.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/no-find-dom-node */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { Button } from '../../Button';
+import { Popover } from '../../Popover';
+import { Overlay } from '../index';
+
+export class MockOverlayManager extends React.Component {
+  constructor() {
+    super();
+    this.buttonRef = React.createRef();
+    this.state = { showOverlay: false };
+  }
+  toggleOverlay = () => {
+    this.setState({ showOverlay: !this.state.showOverlay });
+  };
+  render() {
+    const { placement } = this.props;
+    const { showOverlay } = this.state;
+
+    return (
+      <div>
```

Comment:
optional: you can use `React.Fragment` here instead of `div` so the DOM will be cleaner